output.go

```diff
@@ -117,6 +95,39 @@ func FormatForLink(link *Link) (format Format) {
 	return format
 }
 
+// IsYAMLFile returns true if a given file path corresponds to a YAML file
+func IsYAMLFile(path string) bool {
+	return strings.HasSuffix(path, ".yaml") || strings.HasSuffix(path, ".yml")
+}
+
+// IsTOMLFile returns true if a given file path corresponds to a TOML file
+func IsTOMLFile(path string) bool {
+	return strings.HasSuffix(path, ".toml") || strings.HasSuffix(path, ".tml")
+}
+
+// IsJSONFile returns true if a given file path corresponds to a JSON file
+func IsJSONFile(path string) bool {
+	return strings.HasSuffix(path, ".json")
+}
+
+// IsEnvFile returns true if a given file path corresponds to a .env file
+func IsEnvFile(path string) bool {
+	return strings.HasSuffix(path, ".env")
+}
+
+// IsSimpleValue is intended to see if the underlying value allows a flat map to be retained
+func IsSimpleValue(i interface{}) bool {
+	switch i.(type) {
+	case string,
+		float32, float64,
+		uint8, uint16, uint32, uint64,
```

Comment:
I'm guessing, but I think `int` and `uint` may be their own types not covered by these listed here.

Reply:
good catch!